src/test/groovy/ee/tuleva/onboarding/mandate/batch/MandateBatchFixture.java

```diff
@@ -0,0 +1,8 @@
+package ee.tuleva.onboarding.mandate.batch;
+
+public class MandateBatchFixture {
+
+  public static MandateBatch.MandateBatchBuilder aMandateBatch() {
+    return MandateBatch.builder().status(MandateBatchStatus.INITIALIZED);
```

Comment:
Static import `MandateBatchStatus`.  All of our test fixtures are in groovy currently, is this one in java on purpose?

Reply:
For more compile time safety, it's in Java. 
During large refactors, it's only possible to find out that fixture needs to be updated only when the tests run, not during compile time.